backend/api/v1/endpoints/auth.py

```
@@ -24,59 +22,67 @@
     if db_user:
         raise HTTPException(status_code=400, detail="Username already taken")
-
+    
     hashed_password = hash_password(user.password)
-    new_user = models.User(username=user.username, email=user.email, hashed_password=hashed_password)
-
+    new_user = models.User(
+        username=user.username,
+        email=user.email,
+        hashed_password=hashed_password,
+        profile_completed=False  # ✅ Profile completion starts separately
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)